Mathlib/Data/QPF/Multivariate/Basic.lean

```diff
@@ -67,10 +67,13 @@ matched because they preserve the properties of QPF. The latter example,
    * Const
 
 each proves that some operations on functors preserves the QPF structure
+-/
 
+set_option linter.longLine false in
```

Comment:
I'm a bit torn about this change, but don't feel strongly about this.

Reply:
Is it the segmentation of the doc-module or actually using the `set_option`?

I haven't actually tried, but I think that keeping a single doc-module block and using `set_option` between the `import`s and the doc-module would trigger the "late documentation" linter...

Reply:
It is the segmentation - and probably you're right!